webserver/views/user.py

```diff
@@ -26,6 +27,17 @@ def lastfmscraper(user_id):
     )
     return Response(scraper, content_type="text/javascript")
 
+@user_bp.route("/<user_id>/resettoken", methods=["GET", "POST"])
+@login_required
+def reset_token(user_id):
+    if request.method == "POST":
+        reset = request.form.get("reset")
+        if reset == "yes":
+            db.user.update_token(current_user.id)
+            flash.info("Access token reset")
```

Comment:
What happens if a user submits an invalid token? From the looks of it, it would quietly do nothing. Is that intended due to security?